From: Takashi Iwai <tiwai@suse.de>
Subject: kABI workaround for removed usb_interface.pm_usage_cnt field
Patch-mainline: Never, kABI compatibility for SLE15
References: bsc#1051510

Revive the field that was removed by the recent USB core PM fix.
It's not used any longer but just a place holder.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 include/linux/usb.h |    1 +
 1 file changed, 1 insertion(+)

--- a/include/linux/usb.h
+++ b/include/linux/usb.h
@@ -255,6 +255,7 @@ struct usb_interface {
 
 	struct device dev;		/* interface specific device info */
 	struct device *usb_dev;
+	atomic_t pm_usage_cnt;		/* XXX placeholder only for kABI */
 	struct work_struct reset_ws;	/* for resets in atomic context */
 	void *suse_kabi_padding;
 };
